### PR TITLE
removing-deprecated-cache-code

### PR DESCRIPTION
**Pull request by adirn26**
Fixes: https://github.com/mit-cml/appinventor-sources/issues/1830
Removed depricated DrawingCache functions 
Updated variables accordingly

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by ewpatton** [comment]
@AppInventorWorkerBee ok to test

File: appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java [hunk 1]
```diff
@@ -433,27 +433,19 @@ public CanvasView(Context context) {
...
         int height = getHeight();
-        cache = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888);
-        android.graphics.Canvas c = new android.graphics.Canvas(cache);
+        Bitmap currentBitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888);
+        android.graphics.Canvas c = new android.graphics.Canvas(currentBitmap);
         layout(0, 0, width, height);
         draw(c);
-      }
-      return cache;
+        return currentBitmap;
```
Comment (ewpatton) [inline appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java:443]: Please fix the indentation here. Our code style is to indent a new scope by 2 spaces only.

File: appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java [hunk 2]
```diff
@@ -740,12 +732,12 @@ private int getPixelColor(int x, int y) {
         // If so, maybe we can just draw those sprites instead of building a full
         // cache of the view.
 
-        completeCache = buildCache();
+        completeBitmap = createBitmap();
       }
 
-      // Check the complete cache.
+      // Check the complete bitamp.
```
Comment (ewpatton) [inline appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java:738]: bitamp => bitmap

**Review by ewpatton — CHANGES_REQUESTED** [review]
Two minor things to address.

**Comment by adirn26** [comment]
Fixed Indentation and Typo

**Review by ewpatton — APPROVED** [review]
LGTM, but I need to rebase it onto ucr before merging.

**Pull request merged**